docs/mochitests.md

```diff
@@ -78,6 +78,19 @@ Any of the below APIs that takes a `url` will match it as a substring, meaning t
 * `waitForThreadEvents(dbg, eventName)` - Waits for specific thread events
 * `waitForDispatch(dbg, type)` - Wait for a specific action type to be dispatch. If an async action, will wait for it to be done.
 
+## Pausing
```

Comment:
Do we need a whole section for this? Why not just document the `pauseTest` helper in the API section? Keeping the docs smaller makes it easier to go through, although it's balance.